examples/pod_namespace_watch.py

```diff
@@ -31,6 +31,25 @@ def main():
     v1 = client.CoreV1Api()
     count = 10
     w = watch.Watch()
+    """
+    [INFO: `server-side` & `client-side` timeout]
+
+    `kwargs['timeout_seconds'] = n`:
+       - It is a server-side timeout, i.e. a polite request to the server,
+         mentioning it to cleanly close the connection after n seconds.
+       - If the user have a network outage, then this parameter will do nothing.
+       - The client will hang indefinitely without raising an exception.
+       - You can set this number much higher, maybe to 3600 seconds (1h).
+
+    `kwargs['_request_timeout'] = n`:
+       - It is a client-side timeout, configuring the user's local socket.
+       - If the user have a network outage, resulting into dropping all packets
```

Comment:
ditto, once watch connection is established, server push events to clients, 'user' does not send traffic to server over it